client/components/navbar.jsx

```diff
@@ -104,6 +109,7 @@ export default class Navbar extends React.Component {
               <ul className='drop-down-list'>
                 <li className="sign-up-button" onClick={this.handleSignUp}>Sign Up</li>
                 <li className='sign-in-button' onClick={this.handleSignIn}>Sign In</li>
+                <li className='write-review-button' onClick={this.handleWriteReview}>Write Review</li>
```

Comment:
Instead of doing `onClick` for this, you can put an `<a>` element with an `href` attribute inside of the `li`.